### PR TITLE
Ya photogram industrial 2

### PR DESCRIPTION
**Pull request by CodeX-y**

### CODE REVIEW

**Review by dylansarikas — APPROVED** [review]
Looks great!  Very nice and easy to follow, I learned how to make my own code more readable!!

File: app/models/comment.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#  index_comments_on_photo_id  (photo_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (author_id => users.id)
+#  fk_rails_...  (photo_id => photos.id)
+#
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: "User", counter_cache: true
+  belongs_to :photo
```
Comment (thierrychau) [inline app/models/comment.rb]: can you add a counter_cache?

File: app/models/like.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+#  index_likes_on_photo_id  (photo_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (fan_id => users.id)
+#  fk_rails_...  (photo_id => photos.id)
+#
+class Like < ApplicationRecord
+  belongs_to :fan, class_name: "User", counter_cache: true
+  belongs_to :photo, counter_cache: true
```
Comment (thierrychau) [inline app/models/like.rb:23]: Can you add a validation to make sure a user cannot like the same photo more than once?

**Review by thierrychau — CHANGES_REQUESTED** [review]
Hi Yusra, I have minor comments. Can you look at these and make the edits? Thanks

**Review by thierrychau — APPROVED** [review]
Great job, good to merge! Thanks for the edits

**Pull request merged**